src/styles/styles.js

```diff
@@ -1256,6 +1267,18 @@ const styles = {
         fontSize: variables.iouAmountTextSize,
     }, 0),
 
+    iouPreviewBox: {
+        borderColor: themeColors.border,
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        padding: 15,
+        marginTop: 15,
+    },
+
+    normalRadius: {
```

Comment:
I'm not following exactly why we need this. Border radius should just be part of how we define the button style, not an extra style we need to add.